marimo/_runtime/packages/module_name_to_pypi_name.py

```diff
@@ -0,0 +1,758 @@
+# Copyright 2024 Marimo. All rights reserved.
+# Mapping from a module name to the corresponding package name # on PyPI
+MODULE_NAME_TO_PYPI_NAME = {
```

Comment:
can this be lazy (maybe put under a function)? its not super large but we often won't need to read it